Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -1685,40 +1686,37 @@ - (void)didPressSendButton:(UIButton *)button
 
     text = [text stringByTrimmingCharactersInSet:[NSCharacterSet whitespaceCharacterSet]];
 
-    if (text.length > 0) {
```

Comment:
We weren't consistently doing the same "post-message send" work for all kinds of messages (e.g. playing the "message sent" sound, updating scroll and last visible message state, etc.  I've fixed that.